packages/react-email/src/cli/commands/export.ts

```diff
@@ -62,7 +62,7 @@ export const exportTemplates = async (
       // eslint-disable-next-line
       const component = await import(template);
       // eslint-disable-next-line
-      const rendered = render(component.default({}), options);
+      const rendered = render(component.default.default({}), options);
```

Comment:
Replace the import with a require and I think this will remove the need to do `.default.default`